### PR TITLE
changed branding to popcorn 0.2

### PR DESCRIPTION
**Pull request by SatoCodingStuff**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//popcorn 0.2.x
```
Comment (electron271) [inline src/main.rs]: not needed

**Review by electron271 — CHANGES_REQUESTED** [review]

**Comment by electron271** [comment]
honestly im not a huge fan, i think we should hold off on going to 0.2 for now

other devs thoughts?

**Comment by SatoCodingStuff** [comment]
we did do a rewrite. that why i updated the version.

**Comment by electron271** [comment]
regardless, considering when we did the rewrite we didnt have much code anyway and we are still around the same spot, just less messy code

**Comment by SatoCodingStuff** [comment]
fair. i suppose getting a working cli and them change to 0.2

**Comment by electron271** [comment]
(sorry if i closed prematurely)

**Pull request closed without merging**